tests/console/valgrind.pm

```diff
@@ -146,7 +146,7 @@ sub prepare {
     # Compile the valgrind test program
     assert_script_run 'mkdir -p /var/tmp/valgrind';
     assert_script_run 'cd /var/tmp/valgrind';
-    assert_script_run 'curl -v -o valgrind-test.c ' . data_url('valgrind/valgrind-test.c');
+    script_retry('curl -v -o valgrind-test.c ' . data_url('valgrind/valgrind-test.c'), retry => 10, delay => 30, die => 1);
```

Comment:
Minor comment, IMO retry count=3 could be enough

Reply:
sometimes it does not pass